docs/querying.md

```diff
@@ -2,10 +2,83 @@
 
 The query macro and where clause combine to give you full control over your query.
 
+## Where
+
+Where is using a QueryBuilder that allows you to chain where clauses together to build up a complete query.
+```crystal
+posts = Post.where(published: true, author_id: User.first.id)
+```
+
+It supports different operators:
+```crystal
+Post.where(:created_at, :gt, Time.now - 7.days)
+```
+
+Supported operators are :eq, :gteq, :lteq, :neq, :gt, :lt, :nlt, :ngt, :ltgt, :in, :nin, :like, :nlike
+
+
+This allows you create snippets of clauses that can be easily chained together similar to scopes in Rails.
+```crystal
+class Post < Granite::Base
+...
+  def live
+    where(published: true)
+  end
+
+  def draft
+    where(published: false)
+  end
+
+  def last_week
+    where(:created_at, :gt, Time.now - 7.days)
+  end
+end
+
+posts = Post.live.last_week
+```
+
+## Order
+
+Order is using the QueryBuilder and supports providing an ORDER BY clause:
+```crystal
+Post.order(:created_at)
+```
+
+Direction
+```crystal
+Post.order(updated_at: Sort::Descending)
```

Comment:
I thinks [this can just be `order(updated_at: :desc)`](https://github.com/amberframework/granite/blob/master/src/granite/query/builder.cr#L92-L94), though it gets converted to a Sort::Descending internally